helm/frconfig/templates/config-map.yaml

```diff
@@ -7,4 +7,8 @@ metadata:
 data:
   GIT_REPO: "{{ .Values.git.repo }}"
   GIT_AUTOSAVE_BRANCH:  autosave-{{ .Release.Namespace }}
-  GIT_CHECKOUT_BRANCH:  {{ .Values.git.branch }}
\ No newline at end of file
+  GIT_CHECKOUT_BRANCH:  {{ .Values.git.branch }}
+
+  NAMESPACE: {{ .Release.Namespace }}
```

Comment:
Some charts get the NAMESPACE from the downward API - which is a safer option vs. letting helm set it.

I'd remove NAMESPACE here and set via the api.